torch/_dynamo/config.py

```diff
@@ -251,6 +251,9 @@
     os.environ.get("TORCHDYNAMO_TRANSLATION_VALIDATION_TIMEOUT", "600000")
 )
 
+# Trace through NumPy or graphbreak
+trace_numpy = True
```

Comment:
We had this flag before and it's deleted when upstreaming numpy_torch_interop. Why do we want to add it back?

Reply:
As per the op:
>This was removed in https://github.com/pytorch/pytorch/commit/63d406a6a9502a478eda5cf17601e95e6e896ce3
Restoring, as it's rather useful for debugging.